kernel/selinux/Makefile

```diff
@@ -2,6 +2,14 @@ obj-y += selinux.o
 obj-y += sepolicy.o
 obj-y += rules.o
 
+ifeq ($(shell [ $(VERSION) -lt 5 -o \( $(VERSION) -eq 5 -a $(PATCHLEVEL) -lt 10 \) ]; echo $$?),0)
```

Comment:
I think this is unnecessary, and the shell command is difficult to understand.

Reply:
Reverted in https://github.com/tiann/KernelSU/pull/402/commits/468120cb099b8489d8a157887431a36a733f675d